include/kodlab_mjbots_sdk/mjbots_control_loop.h

```diff
@@ -147,7 +150,8 @@ class MjbotsControlLoop : public AbstractRealtimeObject {
   std::string logging_channel_name_;      /// Channel name to publish logs to, leave empty if not publishing
   lcm::LCM lcm_;                          /// LCM object
   LogClass log_data_;                     /// object containing log data
-  LcmSubscriber<InputClass> lcm_sub_;     /// LCM subscriber object
+  std::shared_ptr<LcmSubscriber> lcm_sub_;     /// LCM subscriber object
+  LcmMessageHandler<InputClass> input_sub_;    /// LCM input subscription
```

Comment:
Naming here is awkward, but fine.

Reply:
I tried to keep with convention, may be worth changing in the future.  For now, I think this is sufficient.